.nais/naiserator-prod.yaml

```diff
@@ -122,7 +122,7 @@ spec:
     - name: SYFOTILGANGSKONTROLL_URL
       value: "https://syfo-tilgangskontroll.prod-fss-pub.nais.io"
     - name: MQGATEWAY_HOSTNAME
-      value: "a01apvl269.adeo.no"
+      value: "mpls02.adeo.no"
```

Comment:
Finnes det noe tilsvarende i preprod, mon tro?

Reply:
Ikke som jeg vet om.

Reply:
Akhtar sier at de ikke har fått satt det opp ennå (bare for en annen gateway enn den vi bruker).